src/cli.js

```diff
@@ -20,9 +20,10 @@ let command = 'node'
 
 const { hasFlag: hasDevFlag } = consumeBoolean('--dev', nomenArgs)
 
-if (hasDevFlag)
-  // tiny chokidar version here
-  command = 'nodemon'
+// With hmr, would we need nodemon?
+// if (hasDevFlag)
+//   // TODO: tinylibs library chokidar
+//   command = 'nodemon'
```

Comment:
We won't have nodemon in the final version, we'd have chokidar monitoring the process and build instead